lib/ansible/modules/monitoring/zabbix/zabbix_user.py

```diff
@@ -608,10 +615,10 @@ def main():
     except Exception as e:
         module.fail_json(msg="Failed to connect to Zabbix server: %s" % e)
 
-    user = User(module, zbx)
+    zbx_api_version = zbx.api_version()[:3]
```

Comment:
Just a cosmetic thing, this probably would have been better done in `__init__` of `User` class as it would eliminate a need to pass another parameter when instancing and object.

Reply:
Thank you for pointed out.  
I think so, too!  
I will fix this code.